pkg/systemd/quadlet/quadlet.go

```diff
@@ -961,6 +963,23 @@ func ConvertKube(kube *parser.UnitFile, isUser bool) (*parser.UnitFile, error) {
 
 	addNetworks(kube, KubeGroup, service, execStart)
 
+	updateMaps := kube.LookupAllStrv(KubeGroup, KeyAutoUpdate)
+	for _, update := range updateMaps {
+		annotation := fmt.Sprintf("--annotation=%s", define.AutoUpdateLabel)
+		updateType := update
+		val := strings.SplitN(update, "/", 2)
+		if len(val) == 2 {
+			annotation = annotation + "/" + val[0]
+			updateType = val[1]
+		}
+		switch updateType {
+		case "local", "registry":
+			execStart.addf("%s=%s", annotation, updateType)
+		default:
+			return nil, fmt.Errorf("invalid %s value: %s", KeyAutoUpdate, update)
```

Comment:
My previous comment is still open (see https://github.com/containers/podman/pull/18975#pullrequestreview-1493469714).  I don't think we should validate input here.  Code scattering comes at a risk.